### ISSUE TITLE
Convert a new corpus to a given dfm matrix?

### ISSUE THREAD

**Issue by xinbinhuang** [quanteda/quanteda#1377]
I am doing topic modeling with the package `topicmodels`. So I new to split the data into train set and test set. I wonder is it possible to transform the test data into a predefined `dfm` object (generated by the training data). 

Thanks

**Comment by koheiw** [comment]
You are advised to post your question to [StackOverflow](https://stackoverflow.com/questions/tagged/quanteda). We respond to questions very quickly.

**Comment by xinbinhuang** [comment]
Thanks, I repost my question on StackOverflow [link](https://stackoverflow.com/questions/50822579/how-to-convert-new-text-data-to-a-predefined-dfm)
